x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_charts/compliance_score_chart.tsx

```diff
@@ -45,6 +46,123 @@ interface ComplianceScoreChartProps {
   onEvalCounterClick: (evaluation: Evaluation) => void;
 }
 
+const CounterButtonLink = ({
+  text,
+  count,
+  color,
+  onClick,
+}: {
+  count: number;
+  text: string;
+  color: EuiTextProps['color'];
+  onClick: EuiLinkButtonProps['onClick'];
+}) => {
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <>
+      <EuiText
+        size="s"
+        style={{
+          fontWeight: euiTheme.font.weight.bold,
+          marginBottom: euiTheme.size.xs,
+        }}
+      >
+        {text}
+      </EuiText>
+
+      <EuiLink
+        color="text"
+        onClick={onClick}
+        css={css`
+          display: flex;
+          &:hover {
+            text-decoration: none;
+          }
+        `}
+      >
+        <EuiText
+          color={color}
+          css={css`
+            &:hover {
+              border-bottom: 2px solid ${color};
+              padding-bottom: 4px;
+            }
+          `}
+          style={{ fontWeight: euiTheme.font.weight.medium, fontSize: '18px' }}
+          size="s"
+        >
+          <CompactFormattedNumber number={count} abbreviateAbove={999} />
+          &nbsp;
+        </EuiText>
+      </EuiLink>
+    </>
+  );
+};
+
+const CompactPercentageLabels = ({
+  onEvalCounterClick,
+  stats,
+}: {
+  onEvalCounterClick: (evaluation: Evaluation) => void;
+  stats: { totalPassed: number; totalFailed: number };
+}) => (
+  <>
+    <CounterLink
+      text="passed"
+      count={stats.totalPassed}
+      color={statusColors.passed}
+      onClick={() => onEvalCounterClick(RULE_PASSED)}
+      tooltipContent={i18n.translate(
+        'xpack.csp.complianceScoreChart.counterLink.passedFindingsTooltip',
+        { defaultMessage: 'Passed findings' }
+      )}
+    />
+    <EuiText size="s">&nbsp;-&nbsp;</EuiText>
+    <CounterLink
+      text="failed"
+      count={stats.totalFailed}
+      color={statusColors.failed}
+      onClick={() => onEvalCounterClick(RULE_FAILED)}
+      tooltipContent={i18n.translate(
+        'xpack.csp.complianceScoreChart.counterButtonLink.failedFindingsTooltip',
+        { defaultMessage: 'Failed findings' }
+      )}
+    />
+  </>
+);
+
+const NonCompactPercentageLabels = ({
```

Comment:
nit: this could be called PercentageLabels instead of NonCompactPercentageLabels